### ISSUE TITLE
Privilege default option in command completion sort order

### ISSUE THREAD

**Issue by hseg** [qutebrowser/qutebrowser#8011]
Noticed when resetting a bunch of bindings -- the suggestions were given in lexical order of the completions (eg `:bind ;I hint images run...` (the custom option) sorted before `:bind ;I hint images tab` (the default)). In particular, the default binding moved around in suggestion order, requiring paying more attention to the command executed.

**Comment by The-Compiler** [comment]
I'm having a hard time trying to understand what you mean. Are you referring to the order of current/default here?

![image](https://github.com/qutebrowser/qutebrowser/assets/625793/68eb1a0e-23da-4ef9-9194-215c4ad71834)

![image](https://github.com/qutebrowser/qutebrowser/assets/625793/8b7e09b6-f0a3-45a0-9301-27b6edfbfd2f)

![image](https://github.com/qutebrowser/qutebrowser/assets/625793/ffd4fa7f-03f5-4b47-8a16-47dfcfc8d612)

(I made up the `zap` part for testing)

**Comment by hseg** [comment]
Yes, that's precisely what I meant. Though, in retrospect, adding `:config-reset` from #2794 would weaken the case for this, and probably be less work.

**Comment by The-Compiler** [comment]
Eh, still makes sense to be consistent here I suppose, indeed alphabetic ordering doesn't make sense there.
